sklearn/preprocessing/_function_transformer.py

```diff
@@ -73,14 +73,51 @@ def __init__(self, func=None, inverse_func=None, validate=True,
         self.inv_kw_args = inv_kw_args
 
     def fit(self, X, y=None):
+        """Fit transformer by checking X.
+
+        If ``validate`` is ``True``, ``X`` will be checked.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+            Input array.
+
+        Returns
+        -------
+        self
+        """
         if self.validate:
             check_array(X, self.accept_sparse)
         return self
 
     def transform(self, X, y=None):
+        """Transform X using the forward function.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+            Input array.
+
+        Returns
+        -------
+        X_out : array-like, shape (n_samples, n_features)
+            Transformed input.
+        """
         return self._transform(X, y, self.func, self.kw_args)
 
     def inverse_transform(self, X, y=None):
+        """Transform X using the inverse function.
```

Comment:
Is this the docstring that we use for inverse? should it not mention something like "it tries to undo the effect of `transform`"?